deploy/pathplanner/paths/1-1SHOT.path

```diff
@@ -33,7 +33,7 @@
   "eventMarkers": [],
   "globalConstraints": {
     "maxVelocity": 4.3,
-    "maxAcceleration": 3.0,
+    "maxAcceleration": 2.5,
```

Comment:
I'm worried these changes are regressions in the plan because they've been tweaked manually in main, but @Cryplo or others would need to confirm.

Reply:
Is it that note detection cannot handle anything greater than 2.5? (That would make sense.)